docs/source/user_guide/configuration_options.rst

```diff
@@ -0,0 +1,890 @@
+==============================
+Receptor Configuration Options
+==============================
+
+---------------------
+Receptor command line
+---------------------
+
+Command line arguments use the following syntax: ``receptor [--<action> [<param>=<value> ...] ...]``
+
+The possible options for ``<action>`` are listed below.  Parameters for actions are listed in their own section.
+
+-------------
+Miscellaneous
+-------------
+
+.. list-table:: Miscellaneous
+    :header-rows: 1
+    :widths: auto
+
+    * - Action
+      - Description
+    * - ``--bash-completion``
+      - Generates a completion script for the Bash shell.
+    * - ``--config <filename>``
+      - Loads additional configuration options from a YAML file.
+    * - ``--control-service``
+      - Runs a control service.
+    * - ``--help``
+      - Show this help
+    * - ``--local-only``
+      - Runs a self-contained node with no backend.
+    * - ``--log-level``
+      - Specifies the verbosity level for command output.
+    * - ``--node``
+      - Specifies the node configuration of this instance.  THis parameter is required.
+    * - ``--trace``
+      - Enables packet tracing output.
+    * - ``--version``
+      - Display the Receptor version.
+
+^^^^^^^^^^^^^^^
+Bash completion
+^^^^^^^^^^^^^^^
+
+To add Receptor auto-completion to the bash session: ``. <(receptor --bash-completion)``
+
+^^^^^^^^^^^^^^^
+Control Service
+^^^^^^^^^^^^^^^
+
+.. list-table:: Control Service
+    :header-rows: 1
+    :widths: auto
+
+    * - Parameter
+      -  Description
+      -  Default value
+    * - ``filename=<string>``
+      - Specifies the filename of a local Unix socket to bind to the service.
+      - No default value.
+    * - ``permissions=<int>``
+      - Socket file permissions
+      - 0600
+    * - ``service=<string>``
+      - Receptor service name to listen on
+      - control
+    * - ``tls=<string>``
+      - Name of TLS server config for the Receptor listener
+      - No default value.
+    * - ``tcplisten=<string>``
+      - Local TCP port or host:port to bind to the control service
+      - No default value.
+    * - ``tcptls=<string>``
+      - Name of TLS server config for the TCP listener
+      - No default value.
+
+^^^^^^^^^
+Log level
+^^^^^^^^^
+
+.. list-table:: Log Level
+    :header-rows: 1
+    :widths: auto
+
+    * - Parameter
+      - Description
+      - Default value
+    * - ``level=<string>``
+      - Log level: Error, Warning, Info or Debug
+      - Error
+
+^^^^
+Node
+^^^^
+
+.. list-table:: Node
+    :header-rows: 1
+    :widths: auto
+
+    * - Parameter
+      - Description
+      - Default value
+    * - ``id=<string>``
+      - Node ID
+      - local hostname
+    * - ``datadir=<string>``
+      - Directory in which to store node data
+      - No default value.
```

Comment:
default is `/tmp/receptor`

Reply:
the default right now without pr #951 is `/tmp/receptor/` 
The second comment was just FYI